service/telemetry_test.go

```diff
@@ -299,6 +368,13 @@ func TestTelemetryInit(t *testing.T) {
 					},
 				}
 			}
+			if tc.otelFormat {
```

Comment:
For the other flags we are explicitly passing them to `newColTelemetry`, should we follow the same pattern here?